modules/040-control-plane-manager/crds/doc-ru-kube_scheduler_webhook_configuration.yaml

```
@@ -0,0 +1,84 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: kubeschedulerwebhookconfigurations.deckhouse.io
+  labels:
+    heritage: deckhouse
+    module: control-plane-manager
+spec:
+  group: deckhouse.io
+  scope: Cluster
+  names:
+    plural: kubeschedulerwebhookconfigurations
+    singular: kubeschedulerwebhookconfiguration
+    kind: KubeSchedulerWebhookConfiguration
+  preserveUnknownFields: false
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          required: ['webhooks']
+          description: |
+            Определяет конфигурацию для включения внешних extender вебхуков для kube-scheduler.
+
+            С их помощью можно гибко настраивать кластер Kubernetes.
+          properties:
+            webhooks:
+              type: array
+              items:
+                type: object
+                required: ['weight', 'clientConfig']
+                properties:
+                  weight:
+                    type: integer
+                    description: |
+                      Числовой множитель для оценок узла, которые генерирует вызов расчета приоритезации.
+
+                      Вес должен быть положительным целым числом.
```

```suggestion
                      Числовой множитель для оценок узла, которые генерирует вызов расчета приоритезации.
```
Типа параметра уже указан